polylogue/models.py

```
@@ -1,43 +1,54 @@
 from __future__ import annotations
 
-from typing import Any, List, Optional
+from copy import deepcopy
+from typing import Any, Dict, List
```

Ruff (0.15.9)

[warning] `typing.Dict` is deprecated, use `dict` instead

(UP035)